tests/unit/test_charm.py

```diff
@@ -114,7 +153,7 @@ def test_config_changed_when_throttle_mode_invalid(self):
 
         self.assertEqual(
             self.harness.model.unit.status.name,
-            BlockedStatus.name,
+            BlockedStatus.name,  # type: ignore
```

Comment:
@gtrkiller did something like this in a similar [case](https://github.com/canonical/nginx-ingress-integrator-operator/pull/53):

```
# Mype can't recognize the name as a string type, so we should skip the type check.
ACTIVE_STATUS_NAME = ActiveStatus.name  # type: ignore[has-type]
...
 assert app_status == ACTIVE_STATUS_NAME
```

Reply:
Okay, I defined the value of `BlovkedStatus.name` as a global variable.